### ISSUE TITLE
this.next missing in requireLogin (p.127)

### ISSUE THREAD

**Issue by Thomas-Lohrum** [DiscoverMeteor/book#348]
Without the else-branch one might get the following console warning "Route dispatch never rendered. Did you forget to call this.next() in an onBeforeAction? "

``` javascript
var requireLogin = function(pause) {
    if (! Meteor.user()) {
        this.render('accessDenied');
        pause();
    } else {
        this.next();
    }
}
```

**Comment by tmeasday** [comment]
Hi @Thomas-Lohrum - are you using the IR 1.0 pre-release by any chance?

**Comment by Thomas-Lohrum** [comment]
Yes, i am on -pre4.

**Comment by Thomas-Lohrum** [comment]
The issue has been resolved in v1.7 of the book. "this.next()" is included now. The issue can be closed.
